docs/configuration.rst

```diff
@@ -281,3 +295,9 @@ For example:
    directory = "chore"
    name = "Other Tasks"
    showcontent = false
+
+   [[tool.towncrier.type]]
+   directory = "auto"
+   name = "Automatic Changes"
```

Comment:
I prefer not to use `auto` here.

Some people might thing that `auto` has special meaning for `towncrier` and that `towncrier` does something automatically.

```suggestion
   [[tool.towncrier.type]]
   directory = "deps"
   name = "Dependency Changes"
```